sui/Cargo.toml

```diff
@@ -33,6 +33,7 @@ serde-name = "0.2.0"
 dirs = "4.0.0"
 clap = { version = "3.1.17", features = ["derive"] }
 telemetry-subscribers = { git = "https://github.com/MystenLabs/mysten-infra", rev = "3b7daedf91fd8937dde26e905b8114cac459b866" }
+prometheus_exporter = "0.8.4"
```

Comment:
I'd prefer use to use the prometheus crate directly and then host the prometheus endpoint using one of the already existing web frameworks we use. I'm hesitant to want to pull in yet another web framework just to host metrics

Reply:
Ok filed https://github.com/MystenLabs/sui/issues/1907